en-US/firefox/new/horizon.lang

```diff
@@ -104,3 +118,28 @@ Freedom is yours
 
 ;Use the only browser built for people, not profit.
 Use the only browser built for people, not profit.
+
+
+## TAG: firefox_new_other_platforms
+;Firefox for other platforms
+Firefox for other platforms
+
+
+## TAG: firefox_new_other_platforms
+;Download Firefox <span>for Windows</span>
+Download Firefox <span>for Windows</span>
+
+
+## TAG: macos_branding
+;Download Firefox <span>for macOS</span>
+Download Firefox <span>for macOS</span>
```

Comment:
By design a lang file can't have the same string twice. The string needs to be there only once, and in the code it's behind the other tag for practical reasons (not the macos_branding one).